clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/Keys.groovy

```diff
@@ -155,6 +183,18 @@ class Keys {
     "${ID}:${Namespace.LOAD_BALANCERS}:${account}:${region}:${loadBalancerId}"
   }
 
+  static String getVipKey(String vipId, String account, String region) {
+    "${ID}:${Namespace.VIPS}:${account}:${region}:${vipId}"
```

Comment:
The keys above are parsing id, account region, but this is doing account, region, id. I don't think that will work.